src/app/pages/online-test/online-test-routing.module.ts

```diff
@@ -1,7 +1,10 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { OnlineTestComponent } from './online-test.component';
 
-const routes: Routes = [];
+const routes: Routes = [{
```

Comment:
we are going to use dynamic router , please remove angular what is hardcoded ,instead use ':id'